### ISSUE TITLE
Nuke redundant pages from docs/migrate firmware docs to MrChromebox's site

### ISSUE THREAD

**Issue by sadlerm4** [chrultrabook/docs#178]
In light of MrChromebox's updated site, it might be a good opportunity to discuss the future direction and content of the docs in order not to unnecessarily duplicate effort.

I suggest that #159 can become a PR on MrChromebox's website repo instead, and perhaps #138 can as well. 

In theory, the entire firmware, unbricking and reverting sections can be removed.

**Comment by ninelore** [comment]
I dont see any reason why the redundancy is a ossue

**Comment by ethanaobrien** [comment]
> I dont see any reason why the redundancy is a ossue

it makes it more difficult to keep up to date when the data is in 2 different places. Nothing is preventing the user from reading the other website either

**Comment by marcsadler** [comment]
> I dont see any reason why the redundancy is a ossue

No one's going to read it if they can just get the information from https://docs.mrchromebox.tech

The opportunity is also there to overhaul the layout of the supported devices table since it does not really need to include firmware-related support info. It's also debatable whether it needs to include Windows support info either, when once again people aren't reading it when they can just go to https://coolstar.org/chromebook/windows.html

**Comment by CoolPenguin27** [comment]
Yeah I agree that it would be better to move the firmware pages to MrChromebox's website since they are needed more on the firmware dedicated website.

**Comment by marcsadler** [comment]
Closing as this is not the direction that the project is considering, which is perfectly fine. Here's to making the docs the best that they can be, and I appreciate all of you for what you do.

**Comment by death7654** [comment]
Why dont we do this? MrChromebox actively maintains the firmware page and we dont have to worry about adding new devicwa

**Comment by MrChromebox** [comment]
> Closing as this is not the direction that the project is considering, which is perfectly fine. Here's to making the docs the best that they can be, and I appreciate all of you for what you do.

I'm not aware of any discussion on here or on Discord which would lead you to believe this

**Comment by ninelore** [comment]
done in #231
